helm/chaos-mesh/templates/controller-manager-rbac.yaml

```diff
@@ -110,9 +110,12 @@ rules:
   - apiGroups: [ "coordination.k8s.io" ]
     resources: [ "leases" ]
     verbs: [ "*" ]
-  - apiGroups: [""]
-    resources: ["configmaps"]
-    verbs: ["*"]
+  - apiGroups: [ "" ]
+    resources: [ "configmaps" ]
+    verbs: [ "*" ]
+  - apiGroups: [ "" ]
+    resources: [ "secrets" ]
+    verbs: [ "get" ]
```

Comment:
these rule about secrets should be added into Role `{{ .Release.Name }}-chaos-controller-manager-target-namespace`, not here.

see L26-L64

Reply:
Fixed.